### ISSUE TITLE
Classic theme : Block Template Part issue not appearing in Appearance 

### ISSUE THREAD

**Issue by ketan-liquiddev** [WordPress/gutenberg#64201]
### Description

Hello Budy,
I am creating classic theme using tailwind but I need to add header and footer using gutenberg.

I added below action in my **functions.php**
`function my_block_template_part_support() {
    add_theme_support( 'block-template-parts' );
}
add_action( 'after_setup_theme', 'my_block_template_part_support' );
`

After added this code Parts section not registering in my admin panel under Appearance.

1. I tested it on more then themes but not worked. my WP version is 6.6
2. tried to disabled all plugins
3. **tested with health/troubleshoot plugin **

Please help me to resolve this issue.

### Step-by-step reproduction instructions

1. Added fresh classic theme
2. After it added block template part action code
3. checked on theme section not showing new menu on appearance 

Reference : https://learn.wordpress.org/tutorial/using-block-template-parts-in-classic-themes/

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by t-hamano** [comment]
Thanks for the report.

In WP6.6, template parts are part of the pattern screen. You can access it from Appearance > Patterns:

![image](https://github.com/user-attachments/assets/dd6068c9-da77-4bca-a530-815132fd1b21)

Please note that in classic themes, this screen only allows you to edit template parts bundled with the theme; you cannot create new template parts or delete them.

**Comment by ketan-liquiddev** [comment]
Thanks for the reply, because I didn't see it on any guide documents so I put the request. I have one an last request 
Can we make the Pattern file dynamic using ACF pro.
I have created custom pattern and I want to add ACF field value on each custom pattern except gutenberg Row html. is this possible.

**Comment by t-hamano** [comment]
> Can we make the Pattern file dynamic using ACF pro

[The Block Bindings API](https://make.wordpress.org/core/2024/03/06/new-feature-the-block-bindings-api/) should help. For example, you can bind any custom field to a paragraph block.

**Comment by ketan-liquiddev** [comment]
Yeah I reviewed before few days ago, but there is some limitations. 
Like I want to add some repeated data like image and title in list via ACF repeater then how we can embed in block.

I used on binding API image and title as simple but getting difficulty on repeated data. for this reason I choose part section using block_template_part but its html format content.

I need to edit real time data correction with some HTML modification

**Comment by t-hamano** [comment]
From my understanding, in the current version of Block Bindings API, you cannot bind to a single block such as an array or object. You may want to ask on the ACF forum.

**Comment by youknowriad** [comment]
Closing as answered above. For the repeater use case, the Block Bindings API doesn't bind arrays yet, so ACF's support forum is the better venue. Thanks!
